ipynb/agp_distance_decay.ipynb

```diff
@@ -0,0 +1,279 @@
+    "install.packages('fields', repos = \"http://cran.rstudio.com/\")\n",
+    "install.packages('dplyr', repos = \"http://cran.rstudio.com/\")\n",
+    "library(biom)\n",
+    "library(phyloseq)\n",
+    "library(qiimer)\n",
+    "library(ggplot2)\n",
+    "library(vegan)\n",
+    "library(fields)\n",
+    "library(dplyr)\n",
+    "\n",
+    "## I was given a 1-column csv file with sample IDs to include in my analyses\n",
+    "use <- read.csv('filepath/file.csv', header = TRUE)\n",
+    "use[, 1] <- as.character(use[, 1])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we'll read in the BIOM table into R for analysis. R really hates non-JSON formatted BIOM tables (e.g., tables generated with macqiime v1.9), so you may have to use BIOM-convert in to reformat to JSON before importing:\n",
+    "\n",
+    "http://biom-format.org/documentation/biom_conversion.html"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "## import the table\n",
+    "biom <- read_biom('filepath/file.biom')\n",
+    "\n",
+    "## subset data by 'Samples To Use' file\n",
+    "dat <- biom_data(biom)\n",
+    "dat <- as.data.frame(as(biom_data(biom), \"matrix\"), header = TRUE)\n",
+    "dat <- dat[, which(names(dat) %in% use[, 1])]\n",
+    "\n",
+    "## subset again by samples that only occur in >= 10 samples per Josh Ladau's instructions\n",
+    "dat <- dat[-c(which(rowSums(dat != 0) < 10)), ]\n",
+    "\n",
+    "## pull out metadata\n",
+    "tax <- as(observation_metadata(biom), \"matrix\")\n",
+    "meta <- as(sample_metadata(biom), \"matrix\")\n",
+    "meta <- as.data.frame(samp)\n",
+    "\n",
+    "## subset metadata by 'use' rows\n",
+    "meta <- meta[c(which(rownames(meta) %in% use[, 1])), ]\n",
+    "\n",
+    "## coordinates matrix for great circle calculation\n",
+    "## I'm assuming column headers weren't renamed, so I'll go with the same for now\n",
+    "coords <- as.matrix(data.frame(lon = as.numeric(as.character(samp$longitude)),\n",
+    "                               lat = as.numeric(as.character(samp$latitude))))\n",
+    "\n",
+    "## normalize biom table\n",
+    "dat.rel <- t(t(dat) / rowSums(t(dat))) \n",
+    "dat.rel <- dat.rel[which(rowSums(dat.rel) > 0), ]\n",
+    "\n",
+    "## cube-root transform normalized abundances for well-behaved residuals\n",
+    "dat.rel.trans <- dat.rel^(1/3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that our data are successfully loaded, let's compute our distance matrices."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "## compute Canberra distances\n",
+    "comm.dist <- vegdist(t(dat.rel.trans), method = 'canberra')\n",
+    "comm.dist <- as.matrix(comm.dist)\n",
+    "\n",
```

Comment:
Based on the notes that came in october, this used to be Bray-Curtis distance. Is there a reason to use Canberra here over Bray-Curtis or UniFrac?